scenes/raster-terrain-offset.yaml

```diff
@@ -0,0 +1,33 @@
+global:
+    sdk_api_key: ''
+    sdk_landcover_api_key: ''
+
+sources:
+  nextzen:
+    type: MVT
+    url: https://tile.nextzen.org/tilezen/vector/v1/256/all/{z}/{x}/{y}.mvt
+    url_params:
+        api_key: global.sdk_api_key
+    rasters: ['terrain']
+  terrain:
+    type: Raster
+    url: https://d3gy1yxv8mxm7h.cloudfront.net/tilezen/landcover/v1/512/all/{z}/{x}/{y}.png
```

Comment:
This seems like it might not age well. Maybe use a Nextzen terrain source or something instead?

Reply:
Sure, I was testing the `sampleRasterAtPixel` fix with this change. Will revert to use nextzen terrain.